ui/apps/platform/src/Containers/Violations/Details/DeploymentDetails.tsx

```diff
@@ -130,6 +150,28 @@ const DeploymentDetails = ({ deployment }) => {
                     <FlexItem>
                         <SecurityContext deployment={relatedDeployment} />
                     </FlexItem>
+                    <FlexItem>
+                        <Title headingLevel="h3" className="pf-u-my-md">
+                            All Network Policies for {deploymentObj.namespace}
```

Comment:
Please stay closer to the UI suggested in https://issues.redhat.com/browse/ROX-9760
I think this should say:

> All Network Policies in "{deploymentObj.namespace}" namespace

Reply:
Done